ooni/nettests/blocking/telegram.py

```diff
@@ -0,0 +1,90 @@
+# -*- encoding: utf-8 -*-
+
+from twisted.internet import defer, reactor
+from twisted.python import usage
+from twisted.internet.endpoints import TCP4ClientEndpoint
+
+from ooni.utils import log
+from ooni.common.tcp_utils import TCPConnectFactory
+from ooni.errors import failureToString
+
+from ooni import nettest
+
+# These are taken from:
+# https://github.com/telegramdesktop/tdesktop/blob/e6d94b5ee7d96a97ee5976dacb87bafd00beac1d/Telegram/SourceFiles/config.h#L205
+TELEGRAM_DCS = [
+    (1, "149.154.175.50", 443),
+    (2, "149.154.167.51", 443),
+    (3, "149.154.175.100", 443),
+    (4, "149.154.167.91", 443),
+    (5, "149.154.171.5", 443)
```

Comment:
Shall I propose to add the IP `91.108.56.165` as it used to be one DC of Telegram --perhaps for older versions?